lumigator/python/mzai/backend/backend/api/routes/health.py

```diff
@@ -1,17 +1,22 @@
 import json
 from http import HTTPStatus
 from uuid import UUID
+from typing import Annotated
+
+import loguru
 
 import loguru
 import requests
 from fastapi import APIRouter, HTTPException
 from schemas.extras import HealthResponse
-from schemas.jobs import JobSubmissionResponse
+from schemas.jobs import JobLogsResponse, JobSubmissionResponse
 
 from backend.settings import settings
 
 router = APIRouter()
 
+def ray_jobs() -> str:
```

Comment:
I'd agree that it might be better suited being composed in backend settings. 

❓ Should`RAY_DASHBOARD_URL` and anything we might add, return a stronger type than `str` (`Path` or `URL`)? 

If it stays, maybe make this 'private' with the underscore? `_get_ray_jobs_url()`

Whichever way it goes I think it would be worth adding a doc string to explain what it should be sending back (including if the trailing slash is expected to be present).

Reply:
Hum. Ok, I'll move into settings, although I see it as a bit api-specific for the moment. I feel that it is growing too much to fit in a single class, but there are no api or service specific property classes right now.

Re: typing, right now the RAY_DASHBOARD_URL is a str, but I agree with moving it into a URL (better than a Path, imho). @veekaybee wdyt?

Ok to commenting, plus I'll take the opportunity to comment some other property in settings too.

Reply:
Generally URLs are passed around as either Paths or strings in Python, and urllib is also fine but if we're looking for a stronger contract, we can use Pydantic's URL types. 

I generally wouldn't recommend Pydantic out of the box, but since we already use Pydantic URL classes, and we have more than a few URLs to parse in settings, we could use them here and constrain the other URLs in settings the same way https://docs.pydantic.dev/2.0/usage/types/urls/

the other benefit is we can pull all the expected elements from the model (from the docs)

```python
from pydantic import (
    BaseModel,
    HttpUrl,
)


class MyModel(BaseModel):
    url: HttpUrl


m = MyModel(url='http://www.example.com')

# the repr() method for a url will display all properties of the url
print(repr(m.url))
#> Url('http://www.example.com/')
print(m.url.scheme)
#> http
print(m.url.host)
#> www.example.com
print(m.url.port)
#> 80
```

Whatever we go with, it would make sense to standardize across the app for legibility.